gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/RunTask.groovy

```diff
@@ -23,9 +23,10 @@ class RunTask {
 
     static JavaExec createRunTask(Project project, String taskName) {
         return project.tasks.create(taskName, JavaExec) { t ->
+            t.dependsOn(project.tasks.jar)
```

Comment:
That would get you the path to the jar, but wouldn't guarantee that the jar exists at task execution time.  This line will tell gradle to ensure the project's jar exists before execution.